assets/pages/digital-subscription-landing/components/digitalSubscriptionLandingHeader.jsx

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+// ----- Imports ----- //
+import React from 'react';
+
+import LeftMarginSection from 'components/leftMarginSection/leftMarginSection';
+import GridPicture from 'components/gridPicture/gridPicture';
+
+import PriceCtaContainer from '../components/priceCtaContainer';
+
+
+// ----- Setup ----- //
+
+const pictureSources = [
+  {
+    gridId: 'digitalSubscriptionHeaderMobile',
+    sizes: '240px',
+    media: '(max-width: 739px)',
+    srcSizes: [342, 684, 1200],
+  },
+  {
+    gridId: 'digitalSubscriptionHeaderTablet',
+    sizes: '407px',
+    media: '(min-width: 740px) and (max-width: 1139px)',
+    srcSizes: [500, 1000, 2000],
+  },
+  {
+    gridId: 'digitalSubscriptionHeaderDesktop',
+    sizes: '809px',
+    media: '(min-width: 1140px)',
+    srcSizes: [500, 1000, 2000, 4045],
+  },
+];
+
+const gridPicture = {
+  sources: pictureSources,
+  fallback: 'digitalSubscriptionHeaderDesktop',
+  fallbackSize: 400,
+  altText: 'digital subscription',
+};
+
+export default function DigitalSubscriptionLandingHeader() {
+  const leftMarginProps = {
+    modifierClasses: ['header-block', 'grey'],
+  };
+  return (
+    <LeftMarginSection {...leftMarginProps}>
+      <CirclesLeft />
+      <CirclesRight />
+      <div className="gridPicture">
+        <GridPicture {...gridPicture} />
+      </div>
+      <div className="headerTitle">
+        <div className="headerTitleCopy">
+          Support The Guardian with a digital subscription
+        </div>
+      </div>
+      <PriceCtaContainer dark />
+    </LeftMarginSection>
+  );
+}
+
+// ----- Aux Components ----- //
+
+function CirclesLeft() {
+  return (
+    <div>
+      <svg
+        className="svg-title-circles--left__mobile"
```

Comment:
This class uniquely describes this SVG, so under BEM I think it would be: `svg-title-circles-left-mobile`? The `--` is to signify a modifier [on top of](http://getbem.com/faq/#block-modifier-mix) an existing class, and `a__b` signifies an element `b` living [inside a block](http://getbem.com/naming/) `a`.